src/consts.ts

```diff
@@ -60,7 +60,7 @@ export const FallbackBondingDuration = 28;
 export const FallbackSessionsPerEra = 6;
 export const FallbackNominatorRewardedPerValidator = 256;
 export const FallbackMaxElectingVoters = 22500;
-export const FallbackExpectedBlockTime = 6000;
+export const FallbackExpectedBlockTime = 1000;
```

Comment:
I think the values above are different for Aleph Zero network, @Marcin-Radecki will know more. It wouldn't matter if they were just fallbacks but we use it (`maxElectingVoters`) so might be worth using the correct defaults right away.